contracts/scope/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "scope"
-version = "1.0.0-beta2"
+version = "1.0.0-beta3"
```

Comment:
Is `1.0.0-beta3` correct here (and the other `Cargo.toml` files)? I see a `1.0.0-beta2` tag and a `release/v1.0.0-beta5` branch, but nothing about `1.0.0-beta3`.

Reply:
I think it's correct because he is cutting a new release of this package.